kong/plugins/jwt/handler.lua

```diff
@@ -12,6 +12,9 @@ local pairs = pairs
 local tostring = tostring
 local re_gmatch = ngx.re.gmatch
 
+local consts = {
+  JWT_CLAIM_HEADER_PREFIX = "X-Jwt-Claim"
+}
```

Comment:
We don't generally create extra tables for local constants.  Please change this to just
```suggestion
JWT_CLAIM_HEADER_PREFIX = "X-Jwt-Claim"
```